cmd/yurt-manager/app/options/options.go

```diff
@@ -24,18 +24,20 @@ import (
 
 // YurtManagerOptions is the main context object for the yurt-manager.
 type YurtManagerOptions struct {
-	Generic            *GenericOptions
-	NodePoolController *NodePoolControllerOptions
-	GatewayController  *GatewayControllerOptions
+	Generic                       *GenericOptions
+	NodePoolController            *NodePoolControllerOptions
+	GatewayController             *GatewayControllerOptions
+	PoolCoordinatorCertController *PoolCoordinatorCertControllerOptions
```

Comment:
I think that all controllers and webhooks in yurt-manager will use this parameter for specifying namespace, so how about add a field `WorkingNamespace` in  field `GenericOptions` instead of adding `PoolCoordinatorCertControllerOptions `?

Reply:
Okay I understand, what I thought before is that the namespace of poordinator can be specified separately, because poordinator and yurt manager are separate components. I will rework these.

Reply:
@y-ykcir Sorry for i didn't make it clear in the issue before, and thanks for your understanding.

Reply:
@rambohe-ch Should I adapt all the components of yurt-manager to `working-namespace` in this pr?

Reply:
@y-ykcir i think that we can add a issue for tracking yurt-manager(you can add if you want to). so you can handle pool-coordinator cert controller in this issue.